packages/usa-in-page-navigation/src/test/in-page-navigation.spec.js

```diff
@@ -175,6 +180,40 @@ tests.forEach(({ name, selector: containerSelector }) => {
       assert(window.scroll.calledOnceWith(sinon.match({ top: 880 })));
     });
 
+    describe("Minimum heading count tests", () => {
+      beforeEach(() => {
+        body.innerHTML = TEMPLATE;
+        theNav = document.querySelector(THE_NAV);
+      });
+
+      afterEach(() => {
+        body.innerHTML = "";
+        window.location.hash = "";
```

Comment:
**issue**: We're not modifying hash, so this isn't needed here.

```suggestion

```

Reply:
Fixed in [d74c91c](https://github.com/uswds/uswds/pull/6205/commits/d74c91c5c2b9f8ac81e764697ed696ee419dc923)